timeseries/src/autogluon/timeseries/trainer/abstract_trainer.py

```diff
@@ -1178,7 +1191,9 @@ def get_model_pred_dict(
         raise_exception_if_failed: bool = True,
         use_cache: bool = True,
         random_seed: Optional[int] = None,
-    ) -> Union[Dict[str, TimeSeriesDataFrame], Tuple[Dict[str, TimeSeriesDataFrame], Dict[str, float]]]:
+    ) -> Union[
+        Dict[str, Optional[TimeSeriesDataFrame]], Tuple[Dict[str, Optional[TimeSeriesDataFrame]], Dict[str, float]]
+    ]:
```

Comment:
Related to the above comment: Should this always be `Tuple[Dict[str, Optional[TimeSeriesDataFrame]], Dict[str, float]]`?